plugins/ps/functions

```diff
@@ -9,8 +9,21 @@ print_dokku_scale_file() {
   done < "$DOKKU_SCALE_FILE"
 }
 
+get_procfile() {
+  local APP="$1"; local IMAGE_TAG="$3"; local IMAGE=$(get_app_image_name $APP $IMAGE_TAG); local DOKKU_PROCFILE="$DOKKU_ROOT/$APP/DOKKU_PROCFILE"
```

Comment:
I wasn't sure about that `$3` either. I got it from the function below that, `generate_scale_file` which had similar input requirements. I'll update to what you have above though.